### ISSUE TITLE
Formatting exception on Inspector.ProceduralAsset.Error

### ISSUE THREAD

**Issue by XDelta** [Yellow-Dog-Man/Resonite-Issues#1430]
### Describe the bug?

On `ProceduralMesh`s 'Mesh Statistics' section, the error state line has a formatting exception so the information is not displayed and instead shows the placeholder `error` text

### To Reproduce

Look at the 'Mesh Statistics' section of any `ProceduralMesh`

### Expected behavior

Error state to be shown correctly

### Screenshots

![afbb785d1c](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/7883807/328dc20c-e5f1-4a70-b3a2-862aa2568dc4)

### Resonite Version Number

2024.3.1.1178

### Log Files

```2:22:11 PM.613 (127 FPS)	Exception formatting message Inspector.ProceduralAsset.Error, Locale Code: en:
System.NullReferenceException: Object reference not set to an instance of an object
  at Jeffijoe.MessageFormat.MessageFormatter.FormatMessage (System.String pattern, System.Collections.Generic.IReadOnlyDictionary`2[TKey,TValue] args, System.Boolean ignoreMissingVariables) [0x0003b] in <ce50cea1308a4384b872fbd8c8ce5938>:0 
  at Elements.Assets.LocaleResource.Format (System.String key, System.Collections.Generic.Dictionary`2[TKey,TValue] arguments) [0x00021] in <68bb33ee65214c599f2a3050729d2d91>:0 
  at FrooxEngine.LocaleResource.Format (System.String key, System.Collections.Generic.Dictionary`2[TKey,TValue] arguments, System.Boolean returnNullIfNotFound) [0x0001b] in <4293c2ca5df04ad4996124d8e04cd024>:0 

  at System.Environment.get_StackTrace () [0x00000] in <9577ac7a62ef43179789031239ba8798>:0 
  at Elements.Core.UniLog.Error (System.String message, System.Boolean stackTrace) [0x00000] in <701a6dd0fec249e7a9385db1a25f32ef>:0 
  at FrooxEngine.LocaleResource.Format (System.String key, System.Collections.Generic.Dictionary`2[TKey,TValue] arguments, System.Boolean returnNullIfNotFound) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.LocaleHelper.SetLocalized (FrooxEngine.IField`1[T] field, System.String key, System.String format, System.Collections.Generic.Dictionary`2[TKey,TValue] arguments) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.LocaleHelper.AssignLocaleString (FrooxEngine.IField`1[T] field, Elements.Core.LocaleString localeString) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.UIX.Text.set_LocaleContent (Elements.Core.LocaleString value) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.UIX.UIBuilder.Text (Elements.Core.LocaleString& text, System.Boolean bestFit, System.Nullable`1[T] alignment, System.Boolean parseRTF, System.String nullContent) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.ProceduralMesh.BuildInspectorUI (FrooxEngine.UIX.UIBuilder ui) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.WorkerInspector.FrooxEngine.WorkerInspector.BuildUIForComponent_Patch1 (FrooxEngine.WorkerInspector , FrooxEngine.Worker , System.Boolean , System.Boolean , System.Predicate`1[T] ) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.WorkerInspector.SetupContainer (FrooxEngine.Worker container, System.Predicate`1[T] memberFilter, System.Predicate`1[T] workerFilter, System.Boolean includeContainer) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.SceneInspector.OnChanges () [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.ComponentBase`1[C].InternalRunApplyChanges (System.Int32 updateIndex) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.UpdateManager.ProcessChange (FrooxEngine.IUpdatable updatable) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.UpdateManager.RunQueue[T] (System.Collections.Generic.Queue`1[T] queue, System.Action`1[T] action) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.UpdateManager.RunChangeApplications () [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.World.FrooxEngine.World.RefreshStep_Patch1 (FrooxEngine.World ) [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.World.Refresh () [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.WorldManager.UpdateStep () [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.WorldManager.RunUpdateLoop () [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.Engine.UpdateStep () [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at FrooxEngine.Engine.RunUpdateLoop () [0x00000] in <4293c2ca5df04ad4996124d8e04cd024>:0 
  at UnityFrooxEngineRunner.FrooxEngineRunner.UpdateFrooxEngine () [0x00000] in <43987ff318374c2d82c6e4ab6f6b99f2>:0 
  at UnityFrooxEngineRunner.FrooxEngineRunner.UnityFrooxEngineRunner.FrooxEngineRunner.Update_Patch1 (UnityFrooxEngineRunner.FrooxEngineRunner ) [0x00000] in <43987ff318374c2d82c6e4ab6f6b99f2>:0 
```

### Reporters

_deltawolf

**Comment by BlueCyro** [comment]
Same issue as #2410 but in a different place, I've also fixed this just now

**Comment by Frooxius** [comment]
Merged fix in 2024.9.13.672, thanks!~
